content_loading/Settings.py

```diff
@@ -45,4 +46,5 @@ class ImageSettings(BaseSettings):
     model_config = SettingsConfigDict(env_prefix='SupportedImage', extra='ignore')
     fileTypes: str
 
-
+class AzureMultiModalModelSettings(AzureOpenAISettings):
+    model_config = SettingsConfigDict(env_prefix='MultiModalAzureOpenAI', extra='ignore')
```

Comment:
Can we make it so that the MultiModalModel settings are optional?  No need to duplicate the settings if they're going to use a multi-modal model for everything anyway.